tool/src/cpp2/header.rs

```diff
@@ -57,7 +47,7 @@ pub struct Header {
     /// typedef struct Foo Foo;
     /// typedef struct Bar Bar;
     /// ```
-    pub forwards: BTreeSet<Forward>,
+    pub forwards: BTreeMap<Option<String>, BTreeSet<Forward>>,
```

Comment:
Nit: Are the keys on this map optional namespaces? If so, please add this to the docs